src/HexagonOffload.cpp

```diff
@@ -607,10 +607,12 @@ class HexagonLinker : public Linker {
         }
 
         static const uint8_t hexagon_plt1[] = {
+            // clang-format off
```

Comment:
What does clang format want to do here? This looks a lot like an idiomatic select pattern (multiple args per line with a trailing comment).